README.md

```diff
@@ -6,6 +6,18 @@
 
 A simple wrapper around the [js-client-sdk](https://github.com/launchdarkly/js-client-sdk) that provides observable feature flags, a ready state to ensure all feature flags are up to date, and routing utilities.
 
+## Flag Types
+LaunchDarkly supports different flag types.
+
+- Boolean flags have two settings: `true` or `false`.
+- Multivariate flags have multiple settings that let you define different variations. Those settings include `string`, `number`, `JSON` `object`, or `JSON array`.
+
+## Use `invertFlag` with flags
+- **Boolean flags**:
+`false` by default. If set to `true`, the inverse of the requiredFlag's value will be used.
+- **Multivariate flags**:
+The flag value can be strings, numbers, JSON objects, or JSON arrays. Therefore, the inverse value of them is `false` , except that the inverse value of `Number(0)` is `true`.  So be careful when using `invertFlag` with Multivariate flags.
+
```

Comment:
What's the inverse of an empty string, object, or array? Are they also `true`?

Reply:
I think we should change `invertFlag` to be `expectedValue` to be more & be less tricky? Considerations up until now were just for boolean flags.

cc: @Gavsum

Reply:
Don't really have much context on the new changes here, but maybe could just make `invertFlag` a non-op for multivariate flags? Also, if the behaviour of invertFlag ends up getting changed it will break this: 
https://github.com/dashhudson/dash-web/blob/b138dfc1cef38a660fda7a3d2e42ab9fb4ad9860/src/app/settings/routes.js#L39

Let me know if that will require an update 👍

Reply:
Given the use case where we introduce `invertFlag` is to invert the Boolean flag value.
Also, using a multi-variate flag won't break this library.
This readme update is just to add a note so that devs know what will happen when they use `invertFlag` and multivariate flag together.

Reply:
I guess I'm mostly just apprehensive to document something that feels like a workaround.

If we don't want to update it now... it's still subject to change in the future when someone needs to assert what value was returned.

Reply:
Yeah, you are right.
My thought is, this is not stating a workaround, just an existing limitation of it. I'm ok with leaving it as it is now.

If we need more support for multivariate in the future, ppl can update the doc at that time.

Reply:
My motivation to update the readme is just to let ppl know what will happen when they use invertFlag and multivariate flag together.
If they need more support for multivariate in the future, they still need to update the code.

Reply:
If this were only internal that would be helpful but I think whereas it is intended to be public we have to be a little more restrictive in what we document.

Reply:
I am ok with leaving it as it is now.

While just thinking ppl use it in the same way for both internal and public. Since for ppl outside of DH, they may run into the case where they use invertFlag and multivariate flag together.